src/item/pickit.py

```diff
@@ -130,7 +130,6 @@ def pick_up_items(self, char: IChar, is_at_trav: bool = False) -> bool:
                         # send log to discord
                         if found_items and closest_item.name not in picked_up_items:
                             Logger.info(f"Picking up: {closest_item.name} ({closest_item.score*100:.1f}% confidence)")
-                            self._game_stats.log_item_pickup(closest_item.name, self._config.items[closest_item.name].pickit_type == 2)
```

Comment:
the pickit doesnt have any more dependency on the game_stats. Can you remove it from the members vars, the constructor and the calls towards it?

Reply:
Good find